src/ribbon/bar.cpp

```diff
@@ -140,7 +140,7 @@ void wxRibbonBar::ShowPanels(bool show)
 void wxRibbonBar::SetWindowStyleFlag(long style)
 {
     m_flags = style;
-    if(m_art)
+    if ( m_art != nullptr )
```

Comment:
I don't think we want to change all the existing `if ( ptr )` to use `if ( ptr != nullptr )` so I'd leave them be.

Reply:
OK, rolling those back...